static-build/utils.tsx

```diff
@@ -56,13 +56,18 @@ export function writeFiles(toOutput: OutputMap) {
   });
 }
 
-export function calculateScore(
-  test: Test,
-  tool: BuildTool,
-): { score: number; possible: number } {
+interface ScoreForToolTest {
+  score: number;
+  possible: number;
+}
+
+export function calculateScore(test: Test, tool: BuildTool): ScoreForToolTest {
   let score = 0;
-  let possible = 1;
+  let possible = 0;
+
   if (test.results[tool]) {
+    possible = 1;
```

Comment:
Aaaaaand here's the actual fix. We were incrementing the 'possible' even for categories.